tdrs-backend/tdpservice/email/email.py

```diff
@@ -43,20 +43,32 @@ def log(msg, logger_context=None, level='info'):
             object_repr=logger_context['object_repr']
         )
 
-
-@shared_task
-def automated_email(email_path, recipient_email, subject, email_context, text_message, logger_context=None):
+def prepare_recipients(recipient_email):
     """
-    Send email to user.
+    Prepare a list of recipients to be emailed.
 
     recipient_email can be either a string (single recipient) or a array of strings.
     """
     recipients = [recipient_email] if type(recipient_email) == str else recipient_email
```

Comment:
Or you could just use recipient_email.extend(a list or a string), in both str or list the response is added as an item to the list.
try:
i = []
i.extend(3)
i.extend([4,5,])